### PR TITLE
Add overlayColor prop to toolTip for users to set the background color

### PR DESCRIPTION
**Pull request by M-Jas**
## What does this PR do?
- Adds a prop to the toolTip to change the overlay background color. If there is a prop to add an overlay, there should be a prop to change the color of that overlay. #1827

## Any background or context you want to provide?
- We are using the tool tip as a floating action button, on an app that our customers can theme(with their branding). Currently the users would only see the default white and this would cause issues. 

- Users can still change the opacity if they want

## Expected vs Actual
- Expected: If I change the color of the overlay color to red the user will see a red background/shadow to the overlay.
- Actual: We(library users) can't change the color so we are stuck with the default white.

## Images & Gifs
Simulator Screen Shot - iPhone XS - 2019-04-18 at 10 40 57

### CODE REVIEW

**Comment by ronffy** [comment]
I need this feature,
I hope the author can handle it as soon as possible.

**Comment by iRoachie** [comment]
Awesome PR 👏 Can you also add this to the typescript definitions?

File: docs/tooltip.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ import { Tooltip, Text } from 'react-native-elements';
 - [`toggleOnPress`](#toggleOnPress)
 - [`width`](#width)
 - [`withOverlay`](#withoverlay)
+- [`overlayColor`](#overlayColor)
```
Comment (iRoachie) [inline docs/tooltip.md]:
```suggestion
- [`overlayColor`](#overlaycolor)
```

**Comment by M-Jas** [comment]
@iRoachie I fixed prop naming per your request. I always forget about the typescript definitions, sorry about that.

**Comment by iRoachie** [comment]
Hey @M-Jas thanks for your work in the last release. We have an opencollective in which developers and companies contribute finances to help push forward the development of this library.

Can you submit an open collective expense of $150? This would've been for your work here as well as https://github.com/react-native-training/react-native-elements/pull/1868.

You can put the description as `Tooltip and SearchBar enhancements for 1.2.0`. Here's the link - https://opencollective.com/react-native-elements/expenses/new

**Pull request merged**